pkgs/desktops/cinnamon/mint-x-icons/default.nix

```diff
@@ -0,0 +1,57 @@
+{ fetchFromGitHub
+, stdenv
+, gnome3
+, gnome-icon-theme
+, hicolor-icon-theme
+, gtk3
+, humanity-icon-theme
+, ubuntu-themes
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mint-x-icons";
+  version = "1.5.5";
```

Comment:
`mint-x-icons` is already packaged in `pkgs/data/icons`. Maybe just move it `pkgs/desktop/cinnamon`.